packages/mindful/marko-web/service.js

```diff
@@ -20,5 +20,26 @@ class MindfulMarkoWebService {
     const post = await this.client.getAdvertisingPostById({ _id }, fragment);
     return post;
   }
+
+  /**
+   * Get post by Id
+   *
+   * @param {string} externalId
+   * @returns {Promise}
+   */
+  async getAdvertisingPostByIdOrImportEntity({
```

Comment:
Change this to `getAdvertisingPostAsNativeStory` with `_id`, `tenant` and `preview` as args. The provider can be hardset to `native-x` and the `type` hard set to `story`.

Use the `convertAdvertisingPostToNativeStory` directly in this method.